docs/zh_cn/tutorials/How to support new model.md

```diff
@@ -0,0 +1,86 @@
+## 如何支持新的模型
```

Comment:
Could you rename file to `how_to_support_new_models.md` with no empty space in filename?